website/versioned_docs/version-5.x/ecosystem-alpinejs.md

```diff
@@ -0,0 +1,236 @@
+---
+id: ecosystem-alpinejs
+title: single-spa-alpinejs
+sidebar_label: AlpineJS
+---
+
+[single-spa-alpinejs](https://github.com/single-spa/single-spa-alpinejs) is a helper library for mounting alpinejs components as
+single-spa applications or parcels.
+
+## Installation
+
+```sh
+npm install --save single-spa-alpinejs
+
+# or
+yarn add single-spa-alpinejs
+```
+
+Alternatively, you can use single-spa-alpinejs from a CDN as a global variable:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/single-spa-alpinejs"></script>
+```
+
+Note that you might want to lock down the package to a specific version. See [here](https://cdn.jsdelivr.net/npm/single-spa-alpinejs) for
+how to do that.
+
+## Usage
+
+- There are three ways the you can define AlpineJS components as single-spa applications or parcels.
+
+### _1 - Template Only_
+
+- The simplest way were the template contains all the required data and initialization logic (including `x-data` and `x-init`) as part of the dom. The template is provided via the options attribute `template`
+
+#### Via npm
+
+```js
+import singleSpaAlpinejs from 'single-spa-alpinejs';
+const alpinejslifecycles = singleSpaAlpinejs({
+  template: `
+    <div class="rounded overflow-hidden shadow-lg font-sans p-1 m-1" 
+         x-data="{ open: false }">
+      <div class="font-bold p-1">Example for x-show attribute</div>
+      <button class="bg-transparent hover:bg-blue-500 text-blue-700 font-semibold 
+              hover:text-white py-2 px-4 border border-blue-500 
+              hover:border-transparent rounded" 
+              @click="open = !open">Open/Close</button>
+      <div x-show="open" class="text-4xl">
+          Hey, I'm open
+      </div>
+    </div>`,
+});
+
+export const bootstrap = alpinejslifecycles.bootstrap;
+export const mount = alpinejslifecycles.mount;
+export const unmount = alpinejslifecycles.unmount;
+```
+
+#### Via cdn
```

Comment:
CDN usage is separate from the three different ways of using it. We should move this section outside of the "Template Only" subheader into its own section.

Reply:
done .. Added just one example showing how to use the CDN version